addon-resizer/go.mod

```diff
@@ -1,6 +1,7 @@
 module k8s.io/autoscaler/addon-resizer
 
 go 1.21
+toolchain go1.22.2
```

Comment:
Should this really be different to the go field? I'm not sure why dependabot would change this.

Reply:
As per [comment](https://github.com/kubernetes/autoscaler/pull/6735#discussion_r1574787319) removing `lgtm` from the PR.

/lgtm cancel